app/controllers/auctions_controller.rb

```diff
@@ -12,7 +12,9 @@ def index
 
   def show
     auction = find_auction
-    @view_model = AuctionShowViewModel.new(auction: auction, current_user: current_user)
+    @view_model = AuctionShowViewModel.new(auction: auction,
```

Comment:
this is a tiny detail, but when I have multiple args like this I usually put each one on its own line (I thought code climate enforced that but I guess not):


```
AuctionShowViewModel.new(
  auction: auction,
  current_user: current_user,
  bid_error: flash[:bid_error]
)
```